README.md

```diff
@@ -322,6 +331,22 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
 - **Notes**:  See this page for their [FAQ's](https://substrate.dev/hacktoberfest/faq/) or this page for their [official announcement](https://www.parity.io/hacktoberfest-2019-parity/).
 
 
+### 4 Interactions
+
+### **QMK** (Limited edition T-shirt and keyboard)
+
+- **Requirements**: There are several ways to earn a shirt, including:
+  - Review/suggest significant changes on a PR with a Hacktoberfest label
+  - Comment on an issue/PR that contributes to the discussion with the hacktoberfest label
+  - Open a PR that documents any new feature that's consistent with our doc style
+  - Open a PR that translates some documentation
+  - Open a PR that fixes a bug in an issue labelled hacktoberfest
+  - Anything that is deemed to be a significant contribution
```

Comment:
Instead of having a list like this, I'd recommend something like "Requirements are listed on their Hacktoberfest site" with an embedded link. It keeps things cleaner. :smile: